.storybook/preview.js

```diff
@@ -56,51 +55,18 @@ const SORT_ORDER = {
 };
 
 export const parameters = {
-  layout: 'centered',
+  viewMode: 'docs',
+  previewTabs: { 'storybook/docs/panel': { index: -1 } },
```

Comment:
This is what actually switches to docs-first. Inspired by Iceland's design system:

- [Source of their preview.js](https://github.com/island-is/island.is/blob/b148c07e6d0f978898ab91f0c58a9d614b2ff439/libs/island-ui/storybook/config/preview.js#L6-L9)
- [Deployed Storybook](https://ui.devland.is/)

I like this solution because we don't have to revisit all our stories, only switch the view mode globally and tweak the nav order for consistency.

🇮🇸